stablehlo/transforms/StablehloLegalizeQuantToInt.cpp

```diff
@@ -48,36 +48,28 @@ limitations under the License.
 namespace mlir::stablehlo {
 namespace {
 
-// TODO: b/311218165 - consider extract this to common utils and better ways to
-// handle polymorphism.
-using QuantType = std::variant<quant::UniformQuantizedType,
-                               quant::UniformQuantizedPerAxisType>;
+using QuantType = quant::QuantizedType;
```

Comment:
IMO not worth aliasing, but up to you.

Reply:
let keep it as is mainly because of the extend it is used in the fine.